hanadb_exporter/exporters/prometheus_exporter.py

```diff
@@ -45,10 +45,10 @@ def _manage_gauge(self, metric, formatted_query_result):
                 # TODO: exception labels not found
                 # TODO: exception value not found
                 try:
-                    labels.insert(metric.labels.index(column_name), column_value)
+                    labels.insert(metric.labels.index(column_name), column_value.lower())
                 except ValueError: # Received data is not a label, check for the value
                     if column_name == metric.value:
-                        metric_value = column_value
+                        metric_value = column_value.lower()
```

Comment:
Careful!
This is the value of the metric, not the name. We don't have to modify this